rmw_fastrtps_cpp/src/namespace_prefix.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NAMESPACE_PREFIX_HPP_
+#define NAMESPACE_PREFIX_HPP_
+
+#include <vector>
+#include <string>
+
+extern "C"
+{
+extern const char * const ros_topic_prefix;
```

Comment:
If you're in an `extern` group, do you need to `extern` the internal definitions? Not sure if this was there previously, but I think the internal `extern`'s should be removed.